### ISSUE TITLE
wrong config in gitignore?

### ISSUE THREAD

**Issue by reiinakano** [CSTR-Edinburgh/merlin#182]
This line in gitignore https://github.com/CSTR-Edinburgh/merlin/blob/master/.gitignore#L11 excludes the following files from being tracked by git:

merlin/misc/scripts/frontend/utils/prepare_txt_done_data_file.py
merlin/src/keras_lib/data_utils.py
merlin/src/work_in_progress/srikanth/bottleneck_scripts/merge_data.py

Idk if this was intentional but you wouldn't be able to commit changes to those file with the current .gitignore

**Comment by ronanki** [comment]
No, it wasn't intentional. We have to fix that or send us a pull request.
